packages/legend-query-builder/src/components/filter/QueryBuilderFilterPanel.tsx

```diff
@@ -872,6 +885,7 @@ const QueryBuilderFilterConditionEditor = observer(
                     isConstant={queryBuilderState.constantState.isValueSpecConstant(
                       node.condition.value,
                     )}
+                    initializeAsEditable={node.isNewlyAdded}
```

Comment:
do you think if it's better to rename `isNewlyAdded` to match `initializeAsEditable`?

Reply:
Hmm, I think `isNewlyAdded` might make more sense, because the property tracks if the node was just added to the filter panel and hasn't been rendered yet. We could use `isNewlyAdded` to control other actions related to a newly added node, not just if it should be initialized as editable, so keeping it more generic with the `isNewlyAdded` name makes more sense to me than making it specific to initializing as editable.